src/core/App/theme.js

```diff
@@ -29,5 +30,8 @@ export const theme = {
     mobileSmall: "10px",
   },
   borderRadiusSmall: "5px",
-  boxShadow: "0px 4px 12px rgba(186, 199, 213, 0.5)",
+  boxShadow: {
+    tile: "0px 4px 12px rgba(186, 199, 213, 0.5)",
+    hover: "0 0 0 2px rgb(140, 194, 255)",
+  },
```

Comment:
jak przerobiłeś `boxShadow` to zadbaj o to żeby wszędzie działało, bo wydaje mi się że na stronie MoviesPage już nie będzie cieni 😅